WycliffeAssociates/8woc
# Bugfix/klappy 1115

## This PR Addresses:
Selections, Quote, edits, Reminders from the same file, were only filtered by reference and occurrence. Now they are filtered by groupId and quote as well.

## To Test This PR:
Open Titus/ImportantWords.
Select a word/phrase in “circumcise” for Titus 1:10. Navigate to “deceive” and the selection should not be there like it would without this PR.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/wycliffeassociates/8woc/344)
